### PR TITLE
fix #73410, #73412, #73414

### LINKED ISSUE #73410: dirilispostasi.com
### Issue URL (Ads)
 [https://www.dirilispostasi.com/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fwww.dirilispostasi.com%2F)
### Comment
Username: @RadicalNoise
> 

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard Browser extension v3.5.31
Browser: | Chrome
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances<br/><br/><b>Language-specific:</b><br/> AdGuard Turkish

### PR DESCRIPTION
**Pull request by wout-codes**
#73410, #73412, #73414

### CODE REVIEW

**Comment by wout-codes** [comment]
@zloyden

File: TurkishFilter/sections/specific.txt [hunk 1]
```diff
@@ -195,6 +195,10 @@ spooor.com##.adv_no2
 spooor.com##.adv-ust
 spooor.com###adv2
 spooor.com##div[id^="banner_"]
+dirilispostasi.com##div[class^="masthead-ad-50"]
```
Comment (zloyden) [inline TurkishFilter/sections/specific.txt:198]: `dirilispostasi.com##div[class^="masthead-ad"]` it's better without `-50`.

File: TurkishFilter/sections/specific.txt [hunk 2]
```diff
@@ -195,6 +195,10 @@ spooor.com##.adv_no2
...
+neokur.com##div[class^="bg_beyaz p_20"]
```
Comment (zloyden) [inline TurkishFilter/sections/specific.txt:199]: Why not `neokur.com###sag_300_div`?

File: TurkishFilter/sections/specific.txt [hunk 3]
```diff
@@ -195,6 +195,10 @@ spooor.com##.adv_no2
...
+memohaber.com##div[class^="tepe-banner"]
+memohaber.com##div[class^="haber-detay-reklam"]
```
Comment (zloyden) [inline TurkishFilter/sections/specific.txt:201]: Those rules are enough here
```
memohaber.com##.haber-detay-reklam
memohaber.com##.tepe-banner
```

**Review by zloyden — CHANGES_REQUESTED** [review]

**Comment by wout-codes** [comment]
Fixed. Why don't you check my other PR? Some contains some important fixes.

**Review by zloyden — APPROVED** [review]

**Comment by zloyden** [comment]
Tbh, @AdguardTeam/filters-maintainers are responsible for reviewing pull requests, not only me.
So, when you create a pull request, you should call @AdguardTeam/filters-maintainers as `Reviewers` and someone of them will take  a look.
Don't worry, our team reviews these requests from time to time.
I also want to thank you for your contribution to our repository. You've done a lot:)

**Pull request merged**